server/main.py

```diff
@@ -94,3 +123,28 @@ async def calculate_spectrum(payload: Payload):
                 "units": spectrum.units[payload.mode],
             },
         }
+
+#[Date] _[database] _[molecule]_[temperature]K_[pressure]atm
+@app.post("/download_spectrum")
+async def download_spec(payload: Payload):
+    print(payload)
+    date = datetime.datetime.now().strftime("%Y_%m_%d_%H_%M_%S");
+    file_name = f"downloaded_spectrum/{date}_{payload.database}_{payload.tgas}k_{payload.pressure}atm.spec"
+    # file_path=f"create_spectrum/{file_name}"
+    print(f".spec file is created here{file_name}")
+   
+    try:
+        spectrum = calculate_spectrum(payload)
+       
+        if payload.use_simulate_slit is True:
+            print("Applying simulate slit")
+            spectrum.apply_slit(payload.simulate_slit, "nm")
+    except radis.misc.warning.EmptyDatabaseError:
+        return {"error": "No line in the specified wavenumber range"}
+    except Exception as exc:
+        print("Error", exc)
+        return {"error": str(exc)}
+    else:
+        spectrum.store(file_name, compress=True, if_exists_then='replace')
+        return FileResponse(file_name, media_type='application/octet-stream', filename=file_name)
```

Comment:
Was black run on this? I think it should use double quotes instead of single quotes. Are we linting the backend?

Reply:
yup! it runs perfectly

Reply:
I don't think we're running lint on the backend. Would you run and commit the output of `black main.py`? It should make all the quotes consistent, etc.
<img width="736" alt="Screen Shot 2022-08-14 at 5 01 08 PM" src="https://user-images.githubusercontent.com/13723264/184554772-1d9600e2-6a15-4815-9185-4d1b0488a5bd.png">